src/test/java/ProjectTest.java

```diff
@@ -0,0 +1,35 @@
+import danielaSCA.Object;
+import danielaSCA.TestLayer;
+import org.testng.annotations.Test;
+import org.apache.http.HttpStatus;
+
+public class ProjectTest {
+
+    @Test
+    public void createProject() {
+        Object project = new Object();
+        project.setName("Testing");
+        project.setColor(42);
+        TestLayer.postStatement("2c5090848fc1b7a3d738a7b8f2e5418be887d510", project)
```

Comment:
the token, project and id should be setup in another object. you can't give those each tiem